sentinelsat/sentinel.py

```diff
@@ -1296,6 +1296,13 @@ def _format_order_by(order_by):
     return ",".join(output)
 
 
+def _safe_parse_gml_footprint(geometry_str):
```

Comment:
You might actually as well merge this `None --> None` check into the `_parse_gml_footprint` function, since it is so generic, and save some code lines that way.

Reply:
Done!